Fixed IPP browse hang (bug #450120).

	(NewPrinterGUI.browse_ipp_queues_thread): Fixed typo (bug
	#450120).
Index: system-config-printer-1.0.0/system-config-printer.py
===================================================================
--- system-config-printer-1.0.0.orig/system-config-printer.py	2008-06-22 18:13:45.035324322 +0200
+++ system-config-printer-1.0.0/system-config-printer.py	2008-06-22 18:14:31.822820219 +0200
@@ -3808,7 +3808,7 @@ class NewPrinterGUI(GtkGUI):
                 title = _("No queues")
                 text = _("There are no queues available.")
 
-            self.show_error_dialog (title, text, self.IPPBrowseDialog)
+            show_error_dialog (title, text, self.IPPBrowseDialog)
             self.IPPBrowseDialog.hide ()
 
         try:
